src/components/menus/ContextMenu/ContextMenu.test.tsx

```diff
@@ -5,10 +5,6 @@ import ContextMenu, { IContextMenuProps, IMenuItem } from './ContextMenu';
 
 const mockOnClick = jest.fn();
 
-jest.mock('nanoid', () => ({
-	nanoid: () => 'a-fixed-random-id',
-}));
-
```

Comment:
We don't need to mock this because we're not taking snapshots. (It was only mocked due to a Jest import error.)

Reply:
Proof that tests still pass without the mock:

```sh
Test Suites: 67 passed, 67 total
Tests:       151 passed, 151 total
Snapshots:   0 total
Time:        20.366 s
Ran all test suites.
✨  Done in 21.89s.
```